romancal/flatfield/tests/test_flatfield.py

```diff
@@ -54,3 +54,38 @@ def test_flatfield_step_interface(instrument, exptype):
     assert (result.data == wfi_image.data).all()
     assert result.var_flat.shape == shape
     assert result.meta.cal_step.flat_field == 'COMPLETE'
+
+
+
+@pytest.mark.parametrize(
+    "instrument, exptype",
+    [
+        ("WFI", "WFI_IMAGE"),
```

Comment:
It won't be easy to add other parameters since the detector, optical element, start/end times, and resulting reference paths are all hard coded in the test below.

Reply:
I can move these to the parameters.. I was just preserving the pattern Perry did above.